tests/ci/integration/mariadb_patch/0004-WIP-Skip-test-that-is-disabled-when-using-OpenSSL-1..patch

```diff
@@ -0,0 +1,29 @@
+From 85cf96e689ef0df5ddefe12cfbcfb18a80e96210 Mon Sep 17 00:00:00 2001
+From: Robin Newhouse <robinnew@amazon.com>
+Date: Mon, 15 May 2023 22:37:36 +0000
+Subject: [PATCH 4/4] WIP: Skip test that is disabled when using OpenSSL 1.1.1
+
+OpenSSL 1.1.1 and AWS-LC use different cipher names than OpenSSL 1.0.2
```

Comment:
Is this implying MariaDB only supports 1.0.2 right now?

Reply:
Nah, I think the original test was just more restrictive. It only allowed OpenSSL1.1.1 to skip the test. This adds both AWS-LC and OpenSSL1.1.1 to skip it.